README.md

```diff
@@ -215,6 +215,35 @@ Then, in `handleFn`, the handler creates a `Schema` object for `CustomClass`,
 adds it to `Components` so that it can be referenced by name `CustomClass`,
 and returns reference to that object.
 
+### Registration configuration
+It is possible to further customize registration by providing custom `RegistrationConfig` to `OpenAPIModelRegistration`.
+
+#### Example
+```scala
+val components = ...
+val registration = OpenAPIModelRegistration(
+  components,
+  config = RegistrationConfig(
+    OpenAPIModelRegistration.RegistrationConfig(
+      sumADTsShape = OpenAPIModelRegistration.RegistrationConfig.SumADTsShape.WithDiscriminator()
+    )
+  )
+)
+```
+
+#### sumADTsShape
+This config property sets how sum ADTs are registered. It has two possible values:
+- `RegistrationConfig.SumADTsShape.WithoutDiscriminator` - default option, doesn't add discriminators
+- `RegistrationConfig.SumADTsShape.WithDiscriminator(discriminatorPropertyNameFn, addDiscriminatorPropertyOnlyToDirectChildren)` - 
```

Comment:
Somehow the readme seems a bit confusing in the sense that the scala example above shows `SumADTsShape.WithDiscriminator()` call without any params, while here, params seem to be expected.

I know that the reason is that both params have default values, but it may not be obvious to the reader right away why the difference. My suggestion:
 - either add a comment to the example above e.g. `// default values apply for discriminatorPropertyNameFn, addDiscriminatorPropertyOnlyToDirectChildren
 - or mention this in this description?

Reply:
Added the comment, let me know whether it looks better now.

Reply:
Yup, it clears up the uncertainty now I'd say.